docs/cudf/source/groupby.md

```diff
@@ -0,0 +1,168 @@
+GroupBy
+=======
+
+cuDF supports a small (but important) subset of
+Pandas' [groupby API](https://pandas.pydata.org/pandas-docs/stable/user_guide/groupby.html).
+
+## Summary of supported operations
+
+1. Grouping by one or more columns
+1. Basic aggregations such as "sum", "mean", etc.
+1. Quantile aggregation
```

Comment:
Can we combine these two and should we be more explicit about what aggregations we support?

Reply:
The table below does that, but I felt `quantile` and `collect` are special enough to be listed explicitly since they're not really documented anywhere else.